cmd/runhcs/container.go

```diff
@@ -489,36 +471,60 @@ func createContainer(cfg *containerConfig) (_ *container, err error) {
 			annotationPreferredRootFSType  = "io.microsoft.virtualmachine.lcow.preferredrootfstype"
 		)
 
-		opts := &uvm.UVMOptions{
-			ID:    vmID(c.ID),
-			Owner: cfg.Owner,
-			// Resources are used for both LCOW/WCOW memory/processor etc.
-			Resources:            c.Spec.Windows.Resources,
-			ConsolePipe:          cfg.VMConsolePipe,
+		bothOpts := &uvm.Options{
+			ID:                   vmID(c.ID),
```

Comment:
We're making these mandatory now, yes? Just that the code to auto-populate them is still in the code base.

Reply:
Not yet. @kevpar is going to make this when he does a followup here.

Reply:
@jhowardmsft what do you mean by mandatory? do we want to always leave ID generation up to the client?

I'm going to work on splitting out some of the default options into a NewOptionsLCOW func. I think we could either generate a default ID (GUID) in that func, or leave it empty and require the client to generate.

Reply:
I don't have particularly strong feelings either way, but I think the direction we're kind of settling on is that it's required for the client to populate things themselves. IOW, mandatory from the perspective of the client pre-populating them.